std/fmt/colors.ts

```diff
@@ -120,9 +120,41 @@ export function white(str: string): string {
 }
 
 export function gray(str: string): string {
+  return brightBlack(str);
+}
+
+export function brightBlack(str: string): string {
```

Comment:
Is change to `gray` intentional?

Reply:
Yes, did not want to remove any identifiers here but gray was returning 90 which is "bright black" so grey becomes an alias of it.

Reply:
For reference, Wikipedia has a pretty color table https://en.wikipedia.org/wiki/ANSI_escape_code#Terminal_output_sequences for reference